webhook/tests/test_models.py

```diff
@@ -0,0 +1,33 @@
+from models import *
+
+
+def test_email_wrapper_normal():
+    api_key = "normalkey123123"
+    email = "some.normal@email.address"
+    obj = EmailWrapper(api_key=api_key, email=email)
+
+    assert obj.api_key == api_key
+    assert obj.email == email
+
+
+def check_value_error(api_key="normalkey123123", email="some@email.address"):
+    try:
+        EmailWrapper(api_key=api_key, email=email)
+    except ValueError:
+        return True
+    except:
+        return False
+
+def test_email_wrapper_bad_api_key():
```

Comment:
Check `pytest.mark.parametrize` (https://docs.pytest.org/en/stable/parametrize.html#pytest-mark-parametrize) to parametrize test functions